x-pack/plugins/observability_solution/dataset_quality/public/hooks/use_degraded_docs_chart.tsx

```diff
@@ -149,10 +157,10 @@ export const useDegradedDocsChart = ({ dataStream }: DegradedDocsChartDeps) => {
           : exploreDataInDiscoverText;
       },
       getHref: async () => {
-        return redirectLinkProps.href;
+        return redirectLinkProps.linkProps.href;
       },
       getIconType(): string | undefined {
-        return 'popout';
+        return 'visTable';
```

Comment:
Changed the `popout` icon as we are not opening the "Open in Lens"  link with target: `_blank` by default.

![image](https://github.com/elastic/kibana/assets/2748376/109485f2-ca4d-4c87-80c4-2e60ab2adc4b)